### DISCUSSION TITLE
chmod files to group abc?

### DISCUSSION THREAD

**Discussion by Saorn** [RandomNinjaAtk/docker-radarr-extended#16] [General]
What's the deal with the autoconfig setting chmod'ing the group to abc?  In my unRAID setup that group doesn't exist so it's throwing errors every time it imports.  The standard for unRAID would be to use group users.  I modified it in the script on my setup but it just gets set back when you push an update.

**Comment by RandomNinjaAtk** [comment]
The base container from linuxserver uses abc as user/group.

**Reply by Saorn** [reply]
![radarr_lio](https://user-images.githubusercontent.com/67175496/193468531-1d1ed343-0378-4b16-bf10-6c809e789346.PNG)
Pulled a fresh LIO from the unRAID CA catalog.  By default they aren't enabling permission changes.

**Reply by Saorn** [reply]
additionally all unRAID dockers normally run as user 99(nobody) and group 100(users), yours included.

**Reply by RandomNinjaAtk** [reply]
Changes were made in the recent version to fix this, it should no longer be an issue.
